hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieRealtimeFileSplit.java

```diff
@@ -30,22 +32,29 @@
  */
 public class HoodieRealtimeFileSplit extends FileSplit implements RealtimeSplit {
 
+  private static final Logger LOG = LogManager.getLogger(HoodieRealtimeFileSplit.class);
+
   private List<String> deltaLogPaths;
 
   private String maxCommitTime;
 
   private String basePath;
 
+  private HoodieVirtualKeyInfo virtualKeyInfoOpt = null;
```

Comment:
I initially had this as Option< HoodieVirtualKeyInfo>, but was not sure if Option is causing any issues and hence removed and tried.